src/main/java/com/conveyal/r5/analyst/broker/Broker.java

```diff
@@ -380,8 +379,27 @@ public void createWorkersInCategory (WorkerCategory category) {
             throw new RuntimeException(e);
         }
 
+        String workerConfigString = cfg.toString();
+        String scriptTemplate;
+
+        // Read in the startup script
```

Comment:
It would be good to have a more extensive comment here explaining why we do this (to avoid having a special Amazon Machine Image, so we can use a stock Amazon Linux image and have it configure itself).